src/main/java/com/lowcd/TCS/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.lowcd.TCS.controller;
+
+import com.lowcd.TCS.model.LoginBO;
+import com.lowcd.TCS.entity.Role;
+import com.lowcd.TCS.entity.User;
+import com.lowcd.TCS.model.UserBO;
+import com.lowcd.TCS.repository.RoleRepository;
+import com.lowcd.TCS.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin
+@RequestMapping("api/v1/user")
+
+
+
+public class UserController {
+
+    @Autowired
+    private UserService userservice;
+    @Autowired
+    private RoleRepository roleRepository;
+
+  @PostMapping(path="/save")
+  public String saveUser(@RequestBody UserBO userBO)
+  {
+
+
+      Role role = roleRepository.findById(userBO.getRoleId()).get();
+      User user1 = new User(
+              userBO.getUserid(),
+              userBO.getName(),
+              userBO.getPassword(),
+              userBO.getEmail(),
+              role
+      );
+
+      String id = userservice.addUser(user1);
```

Comment:
change the logic of `addUser` such that it accepts a `UserBO` object and creates a `User` object in the method and then saves it